src/chef-mover/src/mover_eredis_sup.erl

```diff
@@ -21,7 +21,7 @@ start_link() ->
 init([]) ->
     Host = envy:get(mover, eredis_host, string),
     Port = envy:get(mover, eredis_port, integer),
-    Password = envy:get(mover, eredis_password, "", string),
+    Password = chef_secrets:get(<<"redis_lb">>, <<"password">>),
```

Comment:
this would return `{ok, Password}` on success -- it looks like we'd want to match that here